developer/src/kmc-keyboard-info/src/index.ts

```diff
@@ -164,8 +167,8 @@ export class KeyboardInfoCompiler {
 
     this.fillLanguages(keyboard_info, kmpJsonData);
 
-    // TODO: use: TZ=UTC0 git log -1 --no-merges --date=format:%Y-%m-%dT%H:%M:%SZ --format=%ad
-    keyboard_info.lastModifiedDate = (new Date).toISOString();
+    // If a last commit date is not given, then just use the current time
+    keyboard_info.lastModifiedDate = sources.lastCommitDate ?? (new Date).toISOString();
```

Comment:
Since I was unfamiliar with it:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString

It appears the mentioned ISO 8601 is quite related to RFC 3339, but there are a few distinctions between the two noted by [ISO 8601's Wikipedia page](https://en.wikipedia.org/wiki/ISO_8601).  (CTRL-F: `RFC`)

The main distinction appears to be focused upon specification of timezone data as part of the string, which we're not using... so that sounds fine.  RFC 3339 allows use of spaces to separate date and time, while ISO 8601 doesn't... but it seems like the Git string doesn't use that, based upon `expectedGitDateFormat` below, so that also sounds fine.

Just wanted to make sure that our default here yields something that should be consistently parse-able by an established specification.